WatchParty/backend/config/config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  development: {
+    dialect: 'sqlite',
+    storage: './db.development.sqlite',
+    session_secret: 'keyboard cat',
+    salt_rounds: 10,
+    demo_user_password: 'pass1234',
+  },
+  test: {
```

Comment:
We have a test database on AWS, also not sure how Heroku would react to seeing sqlite. Can we change this to use for example `username: process.env.DB_USERNAME`

Reply:
Sure. I'll change the test config to use env variables like the production env. Something like:

```
test: {
    username: process.env.TEST_DB_USERNAME,
    password: process.env.TEST_DB_PASSWORD,
    database: process.env.TEST_DB_NAME,
    host: process.env.TEST_DB_HOSTNAME,
    dialect: 'postgres',
    use_env_variable: 'TEST_DATABASE_URL',
    session_secret: process.env.TEST_SESSION_SECRET,
    salt_rounds: 10,
}
```

Reply:
Sounds good :) Thank you!